NEWS.md

```diff
@@ -1,3 +1,7 @@
+# sircovid 0.10.22
```

Comment:
can you add news for this version please? and also bump the package version number

Reply:
I'm leaving this for now as it keeps conflicting with my other branches. Will resolve after https://github.com/mrc-ide/sircovid/pull/256 is merged